tests/Organization.py

```diff
@@ -162,6 +162,11 @@ def testCreateTeamWithAllArguments(self):
         self.assertEqual(team.id, 189852)
         self.assertEqual(team.description, "Description also created by PyGithub")
 
+    def testCreateTeamWithParent(self):
```

Comment:
There is a replay data file that is missing here, which is why your test is failing. Did you forget to `git add` it, or have you not generated it?

Reply:
I was having issues generating it. https://github.com/PyGithub/PyGithub/pull/1669#issuecomment-695116291

Reply:
Some of the older tests include references to PRs/repos and so on that don't exist any more. :-( In this case, you can either comment out setUp so doesn't try and fetch it and then fail (and then revert that once your replay data is written), or separate it out to a different test file.